content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,56 @@
+# Add Dependencies or Outputs to a Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of the build requirements are missing from `dvc.yaml`:
+
+- Files or directories in the <abbr>workspace</abbr> that are dependencies of
+  the stage, are missing from `deps` field.
```

Comment:
@jorgeorpinel I have made some changes to add dependencies part. Also updated to run/commit.

Reply:
Sure, reviewing https://github.com/iterative/dvc.org/pull/1913#pullrequestreview-525917714 now. Resolving this. Thanks